fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocksLex.g4

```diff
@@ -254,6 +254,7 @@ PLUGIN: 'PLUGIN';
 PLUGINS: 'PLUGINS';
 PRECEDING: 'PRECEDING';
 PRIMARY: 'PRIMARY';
+PRIVILEGES: 'PRIVILEGES';
```

Comment:
have we used `PRIVILEGES` as a keyword in `StarRocks.g4`?

Reply:
ALL is an abbreviation for ALL PRIVILEGES